File: fix_syntax.py
import re

filepath = 'frontend/app/dashboard/page.js'
with open(filepath, 'r', encoding='utf-8') as f:
    code = f.read()

# We need to find the `)}` right after `Trash2 size={14}` and replace it with `))}`
pattern = r"(<Trash2 size=\{14\} /></button>\s*</>\s*)\)\}"
replacement = r"\1))}"

code = re.sub(pattern, replacement, code)

with open(filepath, 'w', encoding='utf-8') as f:
    f.write(code)

print("Fixed syntax")

File: fix_syntax.js
const fs = require('fs');

['frontend/app/dashboard/page.js', 'frontend/app/login/page.js'].forEach(file => {
  let code = fs.readFileSync(file, 'utf8');
  code = code.replace(/\\'Reactivate\\'/g, "'Reactivate'");
  fs.writeFileSync(file, code);
});

console.log("Fixed syntax error");
